links.html

```diff
@@ -0,0 +1,21 @@
+<a href="http://frodo.hserus.net/mailman/listinfo/ilugd" target="_blank" rel=external>Subscribe</a></li>
```

Comment:
This file doesn't seem to be included in the main site.

Reply:
Ya. Forgot about that. putting them in.......

Reply:
have we progressed any further on this? @axios047